### DISCUSSION TITLE
Badge colour in sidebar2

### DISCUSSION THREAD

**Discussion by curtic00** [vikramlearning/blazorbootstrap#808] [Q&A] [answered]
In the sidebar, the top area where you have the badge text, etc...

<Sidebar2 Href="/"
          IconName="IconName.BootstrapFill"
          Title="Blazor Bootstrap"
          BadgeText="v2.1.0"
          DataProvider="Sidebar2DataProvider" />
          
Is it possible to change the colour from the default blue?

<img width="341" alt="image" src="https://github.com/user-attachments/assets/31347a95-34b0-4c47-ac4b-d4fdf15c3b90">

**Comment by gvreddy04** [comment, marked as answer]
@curtic00 Thank you for using BlazorBootstrap. Here is the example:

## Sample Code

```css
<style>
	:root {
		--bb-sidebar2-width: 270px;
		--bb-sidebar2-collapsed-width: 50px;
		--bb-sidebar2-background-color: rgba(234, 234, 234, 1);
		--bb-sidebar2-top-row-background-color: rgba(0,0,0,0.08);
		--bb-sidebar2-top-row-border-color: rgb(194,192,192);
		--bb-sidebar2-title-text-color: rgb(0,0,0);
		--bb-sidebar2-brand-icon-color: #6f42c1;
		--bb-sidebar2-brand-image-width: 24px;
		--bb-sidebar2-brand-image-height: 24px;
		--bb-sidebar2-title-badge-text-color: rgb(255,255,255);
		--bb-sidebar2-title-badge-background-color: rgba(25,135,84,var(--bs-bg-opacity,1));
		--bb-sidebar2-navbar-toggler-icon-color: rgb(0,0,0);
		--bb-sidebar2-navbar-toggler-background-color: rgba(0,0,0,0.08);
		--bb-sidebar2-content-border-color: rgb(194,192,192);
		--bb-sidebar2-nav-item-text-color: rgba(0,0,0,0.9);
		--bb-sidebar2-nav-item-text-active-color-rgb: 0,0,0;
		--bb-sidebar2-nav-item-text-hover-color: rgba(var(--bb-sidebar-nav-item-text-active-color-rgb),0.9);
		--bb-sidebar2-nav-item-text-active-color: rgba(var(--bb-sidebar-nav-item-text-active-color-rgb),0.9);
		--bb-sidebar2-nav-item-background-hover-color: rgba(var(--bb-sidebar-nav-item-text-active-color-rgb),0.08);
		--bb-sidebar2-nav-item-group-background-color: rgba(var(--bb-sidebar-nav-item-text-active-color-rgb),0.08);
	}
</style>
```

## Screenshot

![image](https://github.com/user-attachments/assets/12e3fd22-3352-49e4-8609-9c6eccae025d)

**Comment by curtic00** [comment]
Thanks vikram

Works once I realised where it needed to go. 

Charlie 

> Charlie Curtis 
> Worthing, West Sussex
> BN133AB.
> MCSD, MCSA, MCRDBA, Azure administrator
> Mobile: 0777 254 5516
> LinkedIn: https://uk.linkedin.com/in/charlieecharlie
